src/main/java/org/broadinstitute/hellbender/tools/gvs/common/SchemaUtils.java

```diff
@@ -61,6 +61,12 @@ public class SchemaUtils {
     public static final String TRANCHE_FILTER_NAME = "filter_name";
     public static final String TRANCHE_MODEL = "model";
 
+    //Ploidy table
+    public static final String CHROMOSOME = "chromosome";
+    public static final String SAMPLE_ID = "sample_id";
+    public static final String GENOTYPE = "genotype";
+    public static final String PLOIDY = "ploidy";
+
```

Comment:
why do we need a ploidy col if we have a genotype col?

Reply:
That's actually an artifact of how the table is created, at this point.  First we scan alt-allele to create the table, and part of that contains the genotype  _that we sampled from alt allele_ that we will use to infer ref ploidy as well as an empty column for what the final ploidy will be.  Then we do a _second_ pass over our table and calculate the correct ploidy based on the genotype column.

It means that the genotype column is actually vestigial once we have the ploidy column, but it didn't seem worth the effort at this point to remove it altogether.  I DO think that the moment we change our code to create the table and calculate this stuff from the beginning during ingest, we'll want to trim out all references to that column.  The current sql is definitely not going to be the final sql!

Reply:
Ok thanks for the explanation, I was hurting my brain trying to figure out what genotype meant here...

And while we're on the subject is there a ticket for making the ploidy table automatically?

Reply:
I'll create that ticket now so we don't forget it